cache-config/t3c-generate/config/config.go

```diff
@@ -84,7 +83,7 @@ func GetCfg() (Cfg, error) {
 	getopt.Parse()
 
 	if *version {
-		fmt.Println(AppName + " v" + Version)
+		log.Errorln(AppName + " v" + Version)
```

Comment:
`t3c-generate` hasn't initialized the loggers yet here.
You could either call `log.Init(os.Stderr, os.Stderr, os.Stderr, os.Stderr, os.Stderr)` immediately before this `log.Errorln` (since it exits anyway)

Or, this one is probably ok to leave as a print. The goal of the loggers here is to make it possible for people to have log parsers, and not have them ever break. Since this message results from a call to `--version`, it should never be part of a production system that needs the result to run thru a log parser.

Reply:
Put back to print.